tools/assetpipelineeditor/mainwindow.cpp

```diff
@@ -492,11 +499,13 @@ void MainWindow::openSettings()
 {
     QSettings settings;
     auto clearColor = settings.value("clearColor", QColor(Qt::gray)).value<QColor>();
+    auto generateTangents = settings.value("generateTangents", false).value<bool>();
 
     SettingsDialog dlg(this);
     dlg.setSelectionColor(m_assetInspector->meshInspector()->selectionColor());
     dlg.setClearColor(clearColor);
     dlg.setDefaultClearColor(settings.value("defaultClearColor", true).toBool());
+    dlg.setGenerateTangents(generateTangents);
```

Comment:
pretty sure you can use `m_generateTangent` here

(clearColour is a special case because you don't always want to use the user provided one)

Reply:
done